packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts

```diff
@@ -0,0 +1,47 @@
+// @group:suite
```

Comment:
Please put the file also under the `suite` folder instead of `dashboard` so this is inline. (I am also not sure this runs on the CI if it does not match). Also rename the file from `New` to `new` as we use lowercase for file names.

Reply:
Moreover please add `.test` to the file name => `new-account-message.test.ts`

Reply:
534889de5464e01a3eb5970394719506dc079b1e done